sdk/keyvault/Microsoft.Azure.KeyVault/tests/TestFramework/Microsoft.Azure.KeyVault.TestFramework.csproj

```diff
@@ -13,5 +13,7 @@
     <PackageReference Include="Microsoft.Azure.Management.ResourceManager" />
     <PackageReference Include="Microsoft.Extensions.Configuration" />
     <PackageReference Include="Microsoft.Extensions.Configuration.Json" />
+
+    <ExcludeFromProjectReferenceToConversion Include="Microsoft.Azure.Graph.RBAC" />
```

Comment:
I had to add a way to opt-out for some references.

Reply:
I wonder if we should go the opposite way and keep Track 2 still as "Azure" but include any other special libraries we would like to covert from package to project?

Reply:
It would require way more manual work to keep up-to-date than this single opt-out.